src/test/java/com/synopsys/integration/detect/battery/docker/Dotnet5Test.java

```diff
@@ -25,7 +25,7 @@ void detectUsesDotnet5() throws IOException {
 
             dockerAssertions.successfulDetectorType("NUGET");
             dockerAssertions.atLeastOneBdioFile();
-            dockerAssertions.logContainsPattern("https://sig-repo.synopsys.com/.*bds-integrations-release/com/synopsys/integration/detect-nuget-inspector/"); // Verify we are using the EXTERNAL artifactory to download the inspector.
+            dockerAssertions.logContainsPattern("https://sig-repo.synopsys.com/.*bds-integrations-release/com/blackduck/integration/detect-nuget-inspector/"); // Verify we are using the EXTERNAL artifactory to download the inspector.
```

Comment:
repo.blackduck here as well? Although this might mean updating the value somewhere additional to get this test to work.

Reply:
Done, thanks! What is the additional location?

Reply:
Looks like it's being taken care of here: https://github.com/blackducksoftware/synopsys-detect/pull/1245/files